src/current/_includes/releases/v23.1/v23.1.9.md

```diff
@@ -0,0 +1,244 @@
+- Fixed a bug that could cause CPU usage to increase over time. [#109298][#109298]
+- Fixed a bug that could cause some rows to be silently skipped during [`IMPORT`](../v23.1/import.html) when a node's import worker failed. [cockroachdb/cockroach#109664][#109664]
+- Fixed a bug in [geospatial](../v23.1/spatial-data-overview.html) queries, where a query filter of the form `ST_Distance(geog1, geog2) > constant`, or `ST_MaxDistance(geom1, geom2) > constant`, where the operator is one of `>`, `<`, `>=`, `<=`, or a filter of the form `ST_Distance(geog1, geog2, false) = 0` may mistakenly evaluate to `true` when one or both of the inputs is `NULL` or an empty [geography](../v23.1/architecture/glossary.html#geography) or [geometry](../v23.1/architecture/glossary.html#geometry). More rows could be returned by the query than expected. [#109395][#109395]
+- Fixed an issue where a [split](../v23.1/architecture/distribution-layer.html#range-splits) can be called on an invalid key that's in the form of `someValidKey.Next()` during [restore](../v23.1/restore.html). This split key can land in the middle of a row with [column families](../v23.1/column-families.html), and thus result in failing SQL queries when querying the restored table. [#109778][#109778]
+- Fixed an issue where a [split](../v23.1/architecture/distribution-layer.html#range-splits) can be called on an invalid key that's in the form of `someValidKey.Next()` during [restore](../v23.1/restore.html) with the [cluster setting](../v23.1/cluster-settings.html) `bulkio.restore.use_simple_import_spans=true`. This split key can land in the middle of a row with [column families](../v23.1/column-families.html), and thus result in failing SQL queries when querying the restored table. [#109939][#109939]
+- A bug has been fixed that caused a flood of requests to refresh [cluster settings](../v23.1/cluster-settings.html) on the [Advanced Debug page](../v23.1/ui-debug-pages.html#configuration). If a user would like to see the effect of a modified cluster setting in [DB Console](../v23.1/ui-overview.html), a page-reload is required. [cockroachdb/cockroach#108125][#108125]
+
+<h3 id="v23-1-9-performance-improvements">Performance improvements</h3>
+
+- [Backups](../v23.1/backup-and-restore-overview.html) no longer perform work proportional to the number of pending [write intents](../v23.1/architecture/transaction-layer.html#write-intents) that they encounter, so they are over 100x faster when encountering long-running, bulk writing transactions. [#105526][#105526]
+- Queries that access [`pg_catalog`](../v23.1/pg-catalog.html) and [`information_schema`](../v23.1/information-schema.html) that perform introspection on other tables in those schemas are now significantly faster. [#109736][#109736]
+
+<h3 id="v23-1-9-build-changes">Build changes</h3>
+
+- Fixed a bug in Cluster UI publishing due to `pnpm` version. [cockroachdb/cockroach#107575][#107575]
+
+<h3 id="v23-1-9-miscellaneous">Miscellaneous</h3>
```

Comment:
I haven't done this in a while, but I think I usually relocate the notes under this heading to other headings.